### DISCUSSION TITLE
Drilldown bug

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#6714] [Unsupported Versions]
Hi,
I have come across an issue where the **left click to drill down** in the alerts page does not work. 

what do I mean by doesn’t work: clicking on the alert does nothing when it used to open a list of options - one of them being DrillDown.

I tried restarting soc using **so-soc-restart** and when that didnt work I tried restarting the whole server.
The problem still occurs,
Does anyone have an idea how to fix it?

**Comment by dougburks** [comment]
Please provide more information.

What browser are you using?

Have you tried shift-reload on the page?

Have you tried clearing browser cache?

Have you tried a different browser?

Can you provide a screenshot where left click is working properly and a screenshot where it's not working properly?

**Reply by yula-21** [reply]
Im using chrome web browser, I have tried other browsers and they don’t work well with the system.

This is how the left click is usually:
![5B79FD29-5F34-4CB8-925A-F74E1A9D52BC](https://user-images.githubusercontent.com/85167019/148675246-14baec15-6751-48de-897b-085174c51793.jpeg)

For me nothing happens when I left click - I see the alerts tab as usual.

**Reply by dougburks** [reply]
What version of Chrome are you using and on what operating system?

Have you tried shift-reload on the page?

Have you tried clearing browser cache?

Where exactly are you left-clicking?

**Reply by yula-21** [reply]
Hi,
My browser version is 70.0.3538.102
I don’t believe the issue is with the browser cause I’ve been able to use this feature in the past.

To answer your question, yes I have tried refreshing the browser and it didn’t help.

I am trying to left click the alert to see all events of the alert as shown here:
(https://docs.securityonion.net/en/2.3/alerts.html)

**Reply by dougburks** [reply]
Google Chrome version 70.0.3538.102? That seems pretty old. The current version should be version 97. Please try upgrading to the latest Google Chrome. Also make sure you don't have any Extensions that could be interfering.
